### ISSUE TITLE
Waiting for the file to be downloaded time out

### ISSUE THREAD

**Issue by IgorLeSouverain** [mimshins/figma-backup#14]
I get timed out even though it's downloaded all files and I can open them on figma.

```  ____ _____  _    ____ _____ 
 / ___|_   _|/ \  |  _ \_   _|
 \___ \ | | / _ \ | |_) || |  
  ___) || |/ ___ \|  _ < | |  
 |____/ |_/_/   \_\_| \_\|_|  
                              

 Starting the backup task...
> Authenticating the bot...
        . Bot successfully logged in.
        . Caching the cookies...
        . Looking for cached cookies...
        . Restoring the cached cookies...
(node:278030) ExperimentalWarning: The Fetch API is an experimental feature. This feature could change at any time
(Use `node --trace-warnings ...` to show where the warning was created)
> Backuping the file(2)...
        . Setting the download behaviour...
        > Saving a local copy of the file(2)...
                . Opening up the figma command palette...
                . Typing down the download command...
                . Execute the download command...
                ERR. Download aborted | Timeout of 60s exceeded.
> Backuping the file(Untitled)...
        . Setting the download behaviour...
        > Saving a local copy of the file(Untitled)...
                . Opening up the figma command palette...
                . Typing down the download command...
                . Execute the download command...
                ERR. Download aborted | Timeout of 60s exceeded.
Backup task finished! (time elapsed: 217s)
> Stopping the bot...```

**Comment by IgorLeSouverain** [comment]
I noticed that small files get timed out and everything's fine with big ones

**Comment by mimshins** [comment]
@IgorTheOverlord 
If that's the case, I know what's wrong.
Gonna fix it, thanks for your feedback 🙏

**Comment by IgorLeSouverain** [comment]
Will there be an option to download projects by team's id in the future?

**Comment by mimshins** [comment]
@IgorTheOverlord 
I don't see any reason why it shouldn't be an option.
I'll work on that as well.

BTW, You can still use this version and ignore the "Fake Timeout" error while I figure out a solution to this problem.

**Comment by mimshins** [comment]
@IgorTheOverlord 
This should be resolved (upgrade to v2.0.1), please confirm so I can close the thread.

**Comment by IgorLeSouverain** [comment]
It works perfectly now with small files, you can close the issue
